docs/source/migration.mdx

```diff
@@ -18,22 +18,15 @@ During Apollo Server 4's alpha, we are actively looking to gather feedback and [
 
 > For a list of all breaking changes, see the [changelog](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md).
 
-## Bumped dependencies
-
-### Node.js
-Apollo Server 4 supports Node.js 14 and later. This includes all [LTS and Current versions at the time of release](https://nodejs.org/en/about/releases/).
-
-If you're using an older version of Node.js, upgrade your runtime before upgrading to Apollo Server 4.
+## Package-level changes
```

Comment:
Let's chat about this in real time.

I was trying to differentiate "the changes that literally everybody has to make because the basic API fundamentally changed" from "also we tweaked a bunch of obscure stuff" fwiw.